src/Creator.tsx

```diff
@@ -145,6 +148,35 @@ export const Creator = ({ NFToupon_Key }: Props) => {
   };
 
   const sendDetails = async () => {
+    if (textAreaValue === '') {
+      setValidation(true);
+      setVisible(false);
```

Comment:
why do we need to make this false?

Reply:
It is set to false there because the modal will not show if description is empty there